lib/fog/compute/proxmox/models/servers.rb

```diff
@@ -56,8 +56,10 @@ def id_valid?(vmid)
         def get(vmid)
           requires :node
           path_params = { node: node, type: type, vmid: vmid }
-          server_data = service.get_server_status path_params
+          server_data = service.get_server_status(path_params)
```

Comment:
I guess, this one is completely unnecssary (path_params) instead of path_params. Please handle () like its already used in the file.